### ISSUE TITLE
x axis increase, decrease wrong coz pressing button in x+ graph not changes

### ISSUE THREAD

**Issue by virt12** [virt12/Python#6]
After pressing x+ value and x-, axis x interval changes but no changes of signals.

**Comment by MeNsaaH** [comment]
Where are the values of x used in the plot function?

**Comment by virt12** [comment]
309-314 lines of code self.x0 and self.x1

**Comment by MeNsaaH** [comment]
After running the example, signal graph changes or am I mistaken

**Comment by virt12** [comment]
Yeah graph changes I sent you a video from example to show what happens when we increasing decreasing x axis interval, (-), (+) t in example app.
